cmd/agent/main.go

```diff
@@ -176,6 +182,9 @@ func NewAgentRunCommand() *cobra.Command {
 	command.Flags().IntVar(&metricsPort, "metrics-port",
 		env.NumWithDefault("ARGOCD_AGENT_METRICS_PORT", cmd.ValidPort, 8181),
 		"Port the metrics server will listen on")
+	command.Flags().StringVar(&keepAlivePingInterval, "keep-alive-ping-interval",
```

Comment:
The cobra framework has flags that take duration strings as arguments (`Duration`, `DurationVar`, etc). Using these might remove the need for parsing the duration string yourself.

Reply:
Updated it, but we have to parse for env variable. PTAL